libs/email-builder/src/lib/messages/request-tag-creation-message.ts

```diff
@@ -0,0 +1,29 @@
+import type { EmailEnvInfo } from '@newsletters-nx/newsletters-data-client';
+import { renderRequestTagCreationMessage } from '../components/RequestTagCreationMessage';
+import { getMessageConfig } from '../message-config';
+import type {
+	MessageAboutNewsletterParams,
+	MessageConfig,
+	MessageContent,
+} from '../types';
+
+export function buildTagCreationRequestMessage(
+	params: MessageAboutNewsletterParams,
+	emailEnvInfo: EmailEnvInfo,
+): { content: MessageContent; messageConfig: MessageConfig } {
+	const { newsletter } = params;
+	const messageConfig = getMessageConfig(
```

Comment:
If we wanted to move email addresses out to configuration, we could use the [config-service](https://github.com/guardian/newsletters-nx/blob/5f4e28320c726b8725404de07963310661f5da82/apps/newsletters-api/src/services/configuration/config-service.ts#L14) - message config could make this call if it were parameterised with the `NewsletterMessageId`

Reply:
That would work fine in the code, but I'm not sure the parameter store is ideal for storing and managing email addresses - if we need a separate value for the comma-separated list fo email addresses for every message type it add an extra maintenance job when setting up a new message type or updating email address (eg - let's say central production change their contact email and we need to go into AWS and change the param for every messages type that goes to them)

S3 would probably be a better fit - we could just have a single JSON file with the config - something like:
```JSON
{
"REQUEST_TAG_SETUP": {
  "prodRecipients":["central-prod@example.com"],
  "prodCC":["newsletters-editorail@example.com"]
},
"REQUEST_BRAZE_SETUP": {
  "prodRecipients":["production-team@example.com"],
  "prodCC":["newsletters-editorail@example.com"]
}
}
```

Reply:
I think the paramter store would work just the same right? there's nothing to stop us having that structure in there - with current implementation, it would need prod and code config options but we can allow an override path in there. 
Anyway, not for this PR